src/ariel-os-rt/src/lib.rs

```diff
@@ -42,7 +42,8 @@ cfg_if::cfg_if! {
 const ISR_STACKSIZE: usize =
     ariel_os_utils::usize_from_env_or!("CONFIG_ISR_STACKSIZE", 8192, "ISR stack size (in bytes)");
 
-#[link_section = ".isr_stack"]
+// SAFETY: a linker script inserts this section in RAM.
```

Comment:
```suggestion
// SAFETY: a linker script inserts this section in RAM; the machine expects … there.
```

minor: That's what happens, but what is the condition for something to be placed in that section, and how is it satisfied?